DataProcessing/QuiverCongressDataDownloader.cs

```diff
@@ -83,20 +87,22 @@ public QuiverCongressDataDownloader(string destinationFolder, string apiKey = nu
         public bool Run()
         {
             var stopwatch = Stopwatch.StartNew();
-            var today = DateTime.UtcNow.Date;
+            var today = DateTime.Today;
```

Comment:
> var today = DateTime.Today;

Depends on the local time zone, versus previous implementation `var today = DateTime.UtcNow.Date;` which was normalized to UTC?